### PR TITLE
[ADD] 방 참가하기 화면 구성

### LINKED ISSUE #46: [ADD] 방 참가하기 화면 구현
## 💡 Issue

방 참가하기 화면 구현
## 📝  todo

- [x] 메인화면 -> 방 참가하기 연결
- [ ] 초대코드 입력뷰
- [ ] 방 참가 팝업
- [ ] 캐릭터 선택뷰

### PR DESCRIPTION
**Pull request by coby5502**
## 🟣 관련 이슈
- close #46

## 🟣 구현/변경 사항 및 이유
- 방 참가화면 구성 및 팝업창

## 🟣 PR Point
- 방 정보확인 팝업창을 띄우는 코드가 괜찮은지 봐주세요.
- 화면 이동을 하는 방법이 여러가지가 있는데, present로 일단 했는데 맞는건가요.

## 🟣 참고 사항

Simulator Screen Shot - iPhone 13 - 2022-06-19 at 22 24 06
Simulator Screen Shot - iPhone 13 - 2022-06-19 at 22 24 18

### CODE REVIEW

File: Manito/Manito/Global/Extension/UIColor+Extension.swift [hunk 1]
```diff
@@ -99,6 +99,16 @@ extension UIColor {
         return UIColor(hex: "#8BC4E7")
     }
     
+    // MARK: - yellow
+    
+    static var yellow: UIColor {
```
Comment (MMMIIIN) [inline Manito/Manito/Global/Extension/UIColor+Extension.swift:104]: p2;

변수명이 냅다 yellow로 되있으면 헷갈릴수도 있을 것 같아요 !

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+import UIKit
+
+import SnapKit
+
+class CheckRoomViewController: BaseViewController {
+    
+    // MARK: - Property
+    
+    private var roomInfoImageView: UIImageView = {
```
Comment (dangsal) [inline Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift:16]: p1;

var 말고 let 으로 사용해도 되지 않을까요?

File: Manito/Manito/Global/Extension/UIColor+Extension.swift (hunk 1 above)
Comment (YoonAh-dev) [reply Manito/Manito/Global/Extension/UIColor+Extension.swift:104]: 요거 피그마에 넣으셨나유?

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        let image = UIImageView()
+        image.image = ImageLiterals.imgEnterRoom
+        image.isUserInteractionEnabled = true
+        return image
+    }()
+    
+    private var roomInfoView = RoomInfoView()
```
Comment (dangsal) [inline Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift:23]: p1;

요곳도요!! var 로 하신 이유가 있으신지..!

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+        button.backgroundColor = .yellow
+        button.makeShadow(color: .shadowYellow, opacity: 1.0, offset: CGSize(width: 0, height: 4), radius: 1)
+        button.layer.cornerRadius = 22
+        return button
+    }()
+    
+    // MARK: - Life Cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
```
Comment (MMMIIIN) [inline Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift:58]: p5;

viewDidLoad에 추가로 넣을 함수가 없다면 지워도 된대요 !
@YoonAh-dev 가 그랬습니다 !

File: Manito/Manito/Global/Extension/UIColor+Extension.swift (hunk 1 above)
Comment (YoonAh-dev) [reply Manito/Manito/Global/Extension/UIColor+Extension.swift:104]: 넣으셨군요. 확인~

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift (hunk 3 above)
Comment (MMMIIIN) [inline Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift:23]: p3;

let으로 해도 되지않을까요 ??

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+        noButton.addTarget(self, action: #selector(didTapNoButton), for: .touchUpInside)
+        yesButton.addTarget(self, action: #selector(didTapYesButton), for: .touchUpInside)
+    }
+    
+    // MARK: - Selectors
+    @objc func didTapNoButton() {
+        dismiss(animated: true, completion: nil)
+    }
+    
+    @objc func didTapYesButton() {
```
Comment (dangsal) [inline Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift:113]: p1;

private 로 변경 부탁드려요!!!

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+        roomInfoImageView.snp.makeConstraints {
+            $0.center.equalToSuperview()
+            $0.leading.trailing.equalToSuperview().inset(24)
+            $0.height.equalTo(roomInfoImageView.snp.width).multipliedBy(1.15)
+        }
+        
+        roomInfoImageView.addSubview(roomInfoView)
+        roomInfoView.snp.makeConstraints {
+            $0.top.equalToSuperview().inset(120)
+            $0.leading.trailing.equalToSuperview()
```
Comment (YoonAh-dev) [inline Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift:72]: p5;

bottom 없어도 괜찮나요?

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+            $0.trailing.equalToSuperview().inset(48)
+        }
+    }
+    
+    // MARK: - Configure
+    override func configUI() {
+        view.backgroundColor = .black.withAlphaComponent(0.7)
+        
+        noButton.addTarget(self, action: #selector(didTapNoButton), for: .touchUpInside)
+        yesButton.addTarget(self, action: #selector(didTapYesButton), for: .touchUpInside)
```
Comment (MMMIIIN) [inline Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift:105]: p2;

property부분에서 추가해도 되지 않을까요 ??

File: Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+    }()
+    
+    private let enterButton: MainButton = {
+        let button = MainButton()
+        button.title = "선택"
+        button.addTarget(self, action: #selector(didTapEnterButton), for: .touchUpInside)
+        return button
+    }()
+    
+
```
Comment (dangsal) [inline Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift:45]: p5;

삭제 부탁이요!!

File: Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+        image.image = ImageLiterals.imgNi
+        return image
+    }()
+    
+    lazy var peopleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "X 8인"
+        label.font = .font(.regular, ofSize: 24)
+        return label
+    }()
```
Comment (MMMIIIN) [inline Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift:27]: p5;

lazy var이여야하는 이유가 있을까요 ??
let이어도 될 것 같네요 !

File: Manito/Manito/Screens/ParticipateRoom/ParticipateRoomViewController.swift [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+    override func configUI() {
+        super.configUI()
+    }
+    
+    // MARK: - Selectors
+    @objc func didTapCloseButton() {
+        dismiss(animated: true, completion: nil)
+    }
+    
+    @objc func didTapNextButton() {
```
Comment (dangsal) [inline Manito/Manito/Screens/ParticipateRoom/ParticipateRoomViewController.swift:83]: p5;

요기도 private입니다!

File: Manito/Manito/Screens/ParticipateRoom/UIComponent/InputInvitedCodeView.swift [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+
+import SnapKit
+
+final class InputInvitedCodeView: UIView {
+    
+    // MARK: - Property
+    private let roomsCodeTextField: UITextField = {
+        let texField = UITextField()
+        let attributes = [
+            NSAttributedString.Key.foregroundColor : UIColor.white,
```
Comment (dangsal) [inline Manito/Manito/Screens/ParticipateRoom/UIComponent/InputInvitedCodeView.swift:18]: p5;

요고 아마 안적어도 디폴트 값일 겁니다!

File: Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+    }
+    
+    // MARK: - func
+    
+    private func render() {
+        self.addSubview(peopleImageView)
+        peopleImageView.snp.makeConstraints {
+            $0.leading.top.bottom.equalToSuperview()
+            $0.width.equalTo(60)
+            $0.height.equalTo(60)
```
Comment (MMMIIIN) [inline Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift:47]: p3;

두개 합쳐도 됩니당 ^_____^

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift [hunk 13]
```diff
@@ -0,0 +1,122 @@
...
+    }
+    
+    // MARK: - Selectors
+    @objc func didTapNoButton() {
+        dismiss(animated: true, completion: nil)
+    }
+    
+    @objc func didTapYesButton() {
+        let storyboard = UIStoryboard(name: "ChooseCharacter", bundle: nil)
+        let ChooseCharacterVC = storyboard.instantiateViewController(identifier: "ChooseCharacterViewController")
```
Comment (YoonAh-dev) [inline Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift:115]: p1;

ChooseCharacterViewController.class 를 쓰면 됩니다!

File: Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+
+import UIKit
+
+import SnapKit
+
+final class PeopleInfoView: UIView {
+    
+    // MARK: - Property
+    
+    private var peopleImageView: UIImageView = {
```
Comment (YoonAh-dev) [inline Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift:16]: p5;

```suggestion
    private var peopleImageView = UIImageView(image: ImageLiterals.imgNi)
```
이렇게 해도 됩니다!

File: Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift (hunk 12 above)
Comment (MMMIIIN) [inline Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift:47]: p5;

높이가 추가로 적용이 되던가요 ??

File: Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+        label.font = .font(.regular, ofSize: 34)
+        return label
+    }()
+    
+    lazy var dateLabel: UILabel = {
+        let label = UILabel()
+        label.text = "2022.06.01 ~ 2022.06.06"
+        label.font = .font(.regular, ofSize: 18)
+        return label
+    }()
```
Comment (MMMIIIN) [inline Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift:28]: p3;

이거도 let으로 해도 될 것 같네요 ??

File: Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+    
+    lazy var peopleInfo = PeopleInfoView()
```
Comment (YoonAh-dev) [inline Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift:30]: p1;

private 붙여주세요!

File: Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift [hunk 17]
```diff
@@ -0,0 +1,64 @@
...
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - func
+    
+    private func render() {
+
```
Comment (YoonAh-dev) [inline Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift:46]: p1;

요거 한 줄 지워주세요!

File: Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift (hunk 16 above)
Comment (MMMIIIN) [inline Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift:30]: p3;

요거도 let으로해도 되지 않나요 ??

File: Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift [hunk 18]
```diff
@@ -0,0 +1,92 @@
...
+    
+    private let subTitleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "당신만의 캐릭터를 정해주세요."
+        label.font = .font(.regular, ofSize: 18)
+        label.textColor = .grey002
+        return label
+    }()
+    
+    lazy var closeButton: UIButton = {
```
Comment (YoonAh-dev) [inline Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift:30]: p5;

요건 private이 아닌 이유가 있을까요?

File: Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift [hunk 19]
```diff
@@ -0,0 +1,64 @@
...
+        
+        self.addSubview(dateLabel)
+        dateLabel.snp.makeConstraints {
+            $0.top.equalTo(self.roomLabel.snp.bottom).offset(8)
+            $0.centerX.equalToSuperview()
+        }
+        
+        self.addSubview(peopleInfo)
+        peopleInfo.snp.makeConstraints {
+            $0.top.equalTo(self.dateLabel.snp.bottom).offset(10)
```
Comment (MMMIIIN) [inline Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift:60]: p5;

self를 쓰는게 더 좋은 방법인가요 ?? @YoonAh-dev

File: Manito/Manito/Screens/Main/MainViewController.swift [hunk 20]
```diff
@@ -132,14 +132,24 @@ class MainViewController: BaseViewController {
...
         
         alert.addAction(createRoom)
         alert.addAction(enterRoom)
         alert.addAction(cancel)
         present(alert, animated: true, completion: nil)
     }
+
+    func presentParticipateRoomViewController() {
+        let storyboard = UIStoryboard(name: "ParticipateRoom", bundle: nil)
+        let ParticipateRoomVC = storyboard.instantiateViewController(identifier: "ParticipateRoomViewController")
```
Comment (YoonAh-dev) [inline Manito/Manito/Screens/Main/MainViewController.swift:146]: p5;

```suggestion
        let ParticipateRoomVC = storyboard.instantiateViewController(identifier: ParticipateRoomViewController.class)
```
이렇게 쓸 수 있어요!

File: Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift [hunk 21]
```diff
@@ -0,0 +1,92 @@
...
+        button.title = "선택"
+        button.addTarget(self, action: #selector(didTapEnterButton), for: .touchUpInside)
+        return button
+    }()
+    
+    
+    // MARK: - Life Cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
```
Comment (MMMIIIN) [inline Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift:49]: p3;

없어도 잘 돌아가게끔 Shin께서 만드셨습니다

File: Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift [hunk 22]
```diff
@@ -0,0 +1,92 @@
...
+            $0.leading.trailing.equalTo(view.safeAreaLayoutGuide).inset(16)
+            $0.bottom.equalTo(view.safeAreaLayoutGuide).inset(57)
+            $0.height.equalTo(60)
+        }
+    }
+    
+    // MARK: - Configure
+    override func configUI() {
+        super.configUI()
+    }
```
Comment (MMMIIIN) [inline Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift:82]: p3;

이거도 없어도 되지 않을까요 ?? @YoonAh-dev 맞나요 ??

File: Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift (hunk 19 above)
Comment (YoonAh-dev) [reply Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift:60]: self. 안써도 괜찮아요!

File: Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift (hunk 22 above)
Comment (YoonAh-dev) [reply Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift:82]: 뒷 배경 색상 때문에 필요해요!

File: Manito/Manito/Screens/ParticipateRoom/UIComponent/InputInvitedCodeView.swift [hunk 23]
```diff
@@ -0,0 +1,50 @@
...
+
+import UIKit
+
+import SnapKit
+
+final class InputInvitedCodeView: UIView {
+    
+    // MARK: - Property
+    private let roomsCodeTextField: UITextField = {
+        let texField = UITextField()
```
Comment (MMMIIIN) [inline Manito/Manito/Screens/ParticipateRoom/UIComponent/InputInvitedCodeView.swift:16]: p3;

텍스 필드... ^______^

File: Manito/Manito/Screens/ParticipateRoom/UIComponent/InputInvitedCodeView.swift [hunk 24]
```diff
@@ -0,0 +1,50 @@
...
+        let attributes = [
+            NSAttributedString.Key.foregroundColor : UIColor.white,
+            NSAttributedString.Key.font : UIFont.font(.regular, ofSize: 18)
+        ]
+        texField.backgroundColor = .subBackgroundGrey
+        texField.attributedPlaceholder = NSAttributedString(string: "초대코드 입력", attributes: attributes)
+
```
Comment (MMMIIIN) [inline Manito/Manito/Screens/ParticipateRoom/UIComponent/InputInvitedCodeView.swift:23]: p5;

빈 줄 삭제해주실 수 있나용 ^______^

Comment (coby5502) [reply Manito/Manito/Screens/ParticipateRoom/UIComponent/InputInvitedCodeView.swift:21]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/ParticipateRoom/UIComponent/InputInvitedCodeView.swift (hunk 23 above)
Comment (coby5502) [reply Manito/Manito/Screens/ParticipateRoom/UIComponent/InputInvitedCodeView.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift (hunk 22 above)
Comment (coby5502) [reply Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift:77]: 후후

File: Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift (hunk 21 above)
Comment (coby5502) [reply Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/Main/MainViewController.swift (hunk 20 above)
Comment (coby5502) [reply Manito/Manito/Screens/Main/MainViewController.swift:146]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift (hunk 19 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift:60]: 글쿤요

File: Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift (hunk 18 above)
Comment (coby5502) [reply Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift (hunk 16 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift (hunk 17 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift:46]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift (hunk 16 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift (hunk 15 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift (hunk 12 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift (hunk 14 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift (hunk 13 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift (hunk 12 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/ParticipateRoom/UIComponent/InputInvitedCodeView.swift (hunk 11 above)
Comment (coby5502) [reply Manito/Manito/Screens/ParticipateRoom/UIComponent/InputInvitedCodeView.swift:18]: 지우니깐 스타일이 조금 달라져서, 좀더 고려해보겠습니다

File: Manito/Manito/Screens/ParticipateRoom/ParticipateRoomViewController.swift (hunk 10 above)
Comment (coby5502) [reply Manito/Manito/Screens/ParticipateRoom/ParticipateRoomViewController.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift (hunk 9 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift (hunk 8 above)
Comment (coby5502) [reply Manito/Manito/Screens/ChooseCharacter/ChooseCharacterViewController.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift (hunk 7 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift (hunk 6 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift (hunk 5 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift (hunk 3 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift (hunk 4 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift (hunk 3 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift (hunk 2 above)
Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/CheckRoomViewController.swift]: [2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!

File: Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift (hunk 19 above)
Comment (MMMIIIN) [reply Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift:60]: self 지워도 될 것 같아요 ! 커밋하신거에 안 지워져있네용 ^______^

Comment (coby5502) [reply Manito/Manito/Screens/CheckRoom/UIComponent/RoomInfoView.swift]: [c455e69](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/c455e69fe7426fa68ea67d536443153d41b82217)
지움!

**Review by MMMIIIN — APPROVED** [review]
케미 멋져 !

File: Manito/Manito/Screens/ParticipateRoom/UIComponent/InputInvitedCodeView.swift (hunk 11 above)
Comment (YoonAh-dev) [reply Manito/Manito/Screens/ParticipateRoom/UIComponent/InputInvitedCodeView.swift:18]: 혹시 저 부분 넣었을 때랑 안넣을 때 각각 뷰에서 어떻게 보이는지 스크린샷 찍어주실 수 있나용?

**Review by YoonAh-dev — APPROVED** [review]
수고하셨습니다~

**Review by dangsal — APPROVED** [review]

File: Manito/Manito/Screens/ParticipateRoom/UIComponent/InputInvitedCodeView.swift (hunk 11 above)
Comment (coby5502) [reply Manito/Manito/Screens/ParticipateRoom/UIComponent/InputInvitedCodeView.swift:18]: - 색상 값 넣었을 때
simulator_screenshot_ED13B1F2-1BEA-4652-B47E-ACAB2AA4B075

- 색상 값 안넣었을 때
simulator_screenshot_0049A874-4E42-4C4F-86B1-3678C13470A5

**Pull request merged**